### ISSUE TITLE
Cochlear nerve aplasia

### ISSUE THREAD

**Issue by lena-tr** [obophenotype/human-phenotype-ontology#10580]
**Preferred term label:** Cochlear nerve aplasia

**Synonyms:** Aplasia of the cochlear nerve, Absent cochlear nerve, Absence of cochlear nerve

**Definition (free text, please give PubMed ID):** A developmental defect causing an absence of the cochlear nerve resulting in sensorineural hearing loss (PMID: 37591623)

**Parent term (use hpo.jax.org/app):** Abnormality of the cochlear nerve [HP:0011396]

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number):** Deafness, autosomal dominant 80 (OMIM: 619274)

**Your nano-attribution (ORCID):** 0009-0007-4569-7461

**Comment by pnrobinson** [comment]
@lena-tr 
Adding new term: 
- ID: HP:6000988
- label: Cochlear nerve aplasia
- parents: [Abnormality of the cochlear nerve (HP:0011396)](https://hpo.jax.org/app/browse/term/HP:0011396)
- pmid: PMID:37591623
- def: A developmental defect causing an absence of the cochlear nerve resulting in sensorineural hearing loss.
- comment: 
- orcid: orcid:0009-0007-4569-7461
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10580
- synonyms:
